www-servers/h2o/files/h2o.logrotate

```diff
@@ -0,0 +1,8 @@
+/var/log/h2o/*.log {
+	missingok
+	delaycompress
+	sharedscripts
+	postrotate
+		/etc/init.d/h2o reload > /dev/null 2>&1 || true
```

Comment:
How about `test -r /run/h2o.pid && kill -HUP $(cat /run/h2o.pid)`? This is how nginx does.